src/ensembl/src/shared/components/layout/components/sidebar-modal/SidebarModal.scss

```diff
@@ -0,0 +1,31 @@
+@import 'src/styles/common';
+
+.wrapper {
+  position: relative;
+  width: 100%;
+  background: $white;
+  display: grid;
+  grid-template-areas: "title close"
+                       "content content";
+  grid-template-columns: 1fr 20px;
+  grid-template-rows: 38px auto;
+}
+
+.title {
+  grid-area: title;
+  font-size: 14px;
+  margin-top: 0;
+}
+
+.content { 
+  grid-area: content;
+  overflow: auto;
+}
+
+.closeButton {
+  grid-area: close;
+  position: absolute;
```

Comment:
Since it's placed in a dedicated grid-area, what's the point of `position: absolute` anymore? Won't justify-self: end and margin-top: 3px (or `position: relative; top: 3px;`) be sufficient?

Reply:
I've changed it to `position: relative; top: 3px;`

Reply:
Ok. After this change:
- there's no need for `z-index: 1;` in the closeButton class
- there's no need for `position: relative` in the `.wrapper` class